data_acquisition/clear_expired_jobs.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import os
+import sys
+import psycopg2
+import getpass
+import logging
+
+url_index = 2
+id_index = 0
+
+# Currently just makes a GET request to the URLs in the DB and returns true if the status code is 200
+# TODO: read HTML for websites to see if they're expired even if the status code is 200
```

Comment:
does this require some sort of NLP?

Reply:
It could definitely be accomplished using NLP but, at least for now, it'll probably be something more like looking for keywords like "sorry" or "unavailable" and lack of buttons, form fields, etc.